src/mobile-app/components/camera.tsx

```diff
@@ -98,12 +93,8 @@ export const Camera = (props: IProps) => {
             const canvas = canvasElement.getContext("2d");
 
             if (canvas) {
-              let size = 0;
-              if (parentRef.current) {
-                size = parentRef.current.getBoundingClientRect().width;
-                canvasElement.height = size;
-                canvasElement.width = size;
-              }
+              canvasElement.height = size;
```

Comment:
This will stretch and squash images - on an iPad we are receiving images at 4:3, though other devices might have more variety in their camera output so it would look very unpleasant to have this much distortion. I don't think the distortion in the thumbnail is as much of an issue, since it's just a thumbnail, but I would definitely try to do a little work here to either determine the native aspect ratio of the device camera and use that scaled to fit the screen, or at least remove the issue for the iPads in the test by using 4:3 for our canvas sizing to match an iPad external camera aspect ratio since that's our current target.